src/control/server/storage/scm/ipmctl_test.go

```diff
@@ -387,38 +388,34 @@ func TestIpmctl_getRegionState(t *testing.T) {
 
 func TestIpmctl_prep(t *testing.T) {
 	verStr := "Intel(R) Optane(TM) Persistent Memory Command Line Interface Version 02.00.00.3825"
-	ndctlNsStr := `[{
-   "dev":"namespace1.0",
-   "mode":"fsdax",
-   "map":"dev",
-   "size":3183575302144,
-   "uuid":"842fc847-28e0-4bb6-8dfc-d24afdba1528",
-   "raw_uuid":"dedb4b28-dc4b-4ccd-b7d1-9bd475c91264",
-   "sector_size":512,
-   "blockdev":"pmem1",
-   "numa_node":1
-}]`
-	ndctl2NsStr := `[{
-   "dev":"namespace1.0",
-   "mode":"fsdax",
-   "map":"dev",
-   "size":3183575302144,
-   "uuid":"842fc847-28e0-4bb6-8dfc-d24afdba1528",
-   "raw_uuid":"dedb4b28-dc4b-4ccd-b7d1-9bd475c91264",
-   "sector_size":512,
-   "blockdev":"pmem1",
-   "numa_node":1
-},{
-   "dev":"namespace0.0",
-   "mode":"fsdax",
-   "map":"dev",
-   "size":3183575302144,
-   "uuid":"842fc847-28e0-4bb6-8dfc-d24afdba1529",
-   "raw_uuid":"dedb4b28-dc4b-4ccd-b7d1-9bd475c91265",
-   "sector_size":512,
-   "blockdev":"pmem0",
-   "numa_node":0
-}]`
+
+	genNsJSON := func(numa, nsNum, size, id string) string {
+		nsName := fmt.Sprintf("%s.%s", numa, nsNum)
+		bdName := numa
+		if nsNum != "0" {
+			bdName = nsName
+		}
+		sizeBytes, err := humanize.ParseBytes(size)
+		if err != nil {
+			t.Fatal(err)
+		}
+		return fmt.Sprintf("{\"dev\":\"namespace%s\",\"mode\":\"fsdax\",\"map\":\"dev\","+
+			"\"size\":%d,\"uuid\":\"842fc847-28e0-4bb6-8dfc-d24afdba15%s\","+
+			"\"raw_uuid\":\"dedb4b28-dc4b-4ccd-b7d1-9bd475c91264\",\"sector_size\":512,"+
+			"\"blockdev\":\"pmem%s\",\"numa_node\":%s}", nsName, sizeBytes, id, bdName, numa)
```

Comment:
Could probably be simplified with using raw string with something like this:
```suggestion
		return fmt.Sprintf(`{"dev":"namespace%s","mode":"fsdax","map":"dev",`+
                	`"size":%d,"uuid":"842fc847-28e0-4bb6-8dfc-d24afdba15%s",`+
                	`"raw_uuid":"dedb4b28-dc4b-4ccd-b7d1-9bd475c91264","sector_size":512,`+
                	`"blockdev":"pmem%s","numa_node":%s}`, nsName, sizeBytes, id, bdName, numa)
```

Reply:
will change if I have to re-push